recipe/meta.yaml

```diff
@@ -38,6 +38,7 @@ outputs:
         - python-constraint
         - cachetools
         - cached-property
+        - typing_extensions
       run_constrained:
         - openforcefield ==9999999999
         - importlib_metadata >=4.10 # https://github.com/conda-forge/openff-toolkit-feedstock/pull/37#issuecomment-1314260590
```

Comment:
This can probably be dropped at this point but I can't imagine much harm in keeping it here